validator/sawtooth_validator/execution/processor_iterator.py

```diff
@@ -112,9 +123,30 @@ def remove(self, processor_identity):
     def __repr__(self):
         return ",".join([repr(k) for k in self._processors.keys()])
 
-    def wait_to_process(self, item):
+    def cancellable_wait(self, processor_type, cancelled_event):
+        """Waits for a particular processor type to register or until
```

Comment:
Text wrapping/indentation is odd with this comment, and there should be an empty line prior to Args:.